### PR TITLE
[Onboard] add new sample of Office Add-in

### PR DESCRIPTION
**Pull request by hermanwenhe**

### CODE REVIEW

File: hello-world-office-addin/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+### Edit the manifest
+
+You can find the app manifest in `./appPackage` folder. The folder contains one manifest file:
+
+- `manifest.json`: Manifest file for Office add-in running locally.
+
+You may add any extra properties or permissions you require to this file. See the [schema reference](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema) for more information.
+
+### Deploy and publish
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:57]:
```suggestion

```

File: hello-world-office-addin/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Currently unified manifest of Office add-in is in public preview, so it does not support deploy and publish.
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:59]:
```suggestion

```

File: hello-world-office-addin/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teams-toolkit-cli)
+
+## Minimal path to awesome
+
+### Debug Office add-in
+
+- Please note that the same M365 account should be used both in Teams Toolkit and Office Applications (Word, Excel, Powerpoint, Outlook).
+- From Visual Studio Code only: use the `Run and Debug Activity Panel` in Visual Studio Code, select `Debug in Excel Desktop (Edge Chromium)`, and click the `Run and Debug` green arrow button. Please run VSCode as administrator if localhost loopback for Microsoft Edge Webview hasn't been enabled. Once enbaled, administrator priviledge is no longer required.
+
+   ![Visual Studio Code debug configuration for Office](./assets/launch.png)
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:39]:
```suggestion

```
Seems the image does not exist

File: hello-world-office-addin/README.md (hunk 2 above)
Comment (MingjiaLiu1995) [reply hello-world-office-addin/README.md:59]: put this at front

**Review by MingjiaLiu1995 — CHANGES_REQUESTED** [review]

**Comment by MingjiaLiu1995** [comment]
Hi @summzhan , do we have a content review process for adding new samples to Teams Toolkit? Should we reach out to some reviewers or should we ensure the content ourselves? Thanks!

File: hello-world-office-addin/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+   Once the Office app is open, you can select the ribbon command to show the add-in task pane. The snapshot shows Excel as an example, and the other Office applications share the same behavior.
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:44]:
```suggestion
   Once the Office app is open, you can select the ribbon command to show the add-in task pane. 
```

File: hello-world-office-addin/README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+- Please note that the same M365 account should be used both in Teams Toolkit and Office Applications (Word, Excel, Powerpoint, Outlook).
+- From Visual Studio Code only: use the `Run and Debug Activity Panel` in Visual Studio Code, select `Debug in Excel Desktop (Edge Chromium)`, and click the `Run and Debug` green arrow button. Please run VSCode as administrator if localhost loopback for Microsoft Edge Webview hasn't been enabled. Once enbaled, administrator priviledge is no longer required.
+
+   ![Visual Studio Code debug configuration for Office](./assets/launch.png)
+
+   Once the Office app is open, you can select the ribbon command to show the add-in task pane. The snapshot shows Excel as an example, and the other Office applications share the same behavior. 
+   ![Excel add-in show taskpane](./assets/excel-ribbon.png)
+
+   The taskpane opens and you can check the sideloaded add-in.
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:47]:
```suggestion
- The taskpane opens and you can check the sideloaded add-in.
```

File: hello-world-office-addin/README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+### Debug Office add-in
+
+- Please note that the same M365 account should be used both in Teams Toolkit and Office Applications (Word, Excel, Powerpoint, Outlook).
+- From Visual Studio Code only: use the `Run and Debug Activity Panel` in Visual Studio Code, select `Debug in Excel Desktop (Edge Chromium)`, and click the `Run and Debug` green arrow button. Please run VSCode as administrator if localhost loopback for Microsoft Edge Webview hasn't been enabled. Once enbaled, administrator priviledge is no longer required.
+
+   ![Visual Studio Code debug configuration for Office](./assets/launch.png)
+
+   Once the Office app is open, you can select the ribbon command to show the add-in task pane. The snapshot shows Excel as an example, and the other Office applications share the same behavior. 
+   ![Excel add-in show taskpane](./assets/excel-ribbon.png)
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:45]: Same comment here

File: hello-world-office-addin/README.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+   The taskpane opens and you can check the sideloaded add-in.
+   ![Excel add-in task pane opened](./assets/thumbnail.png)
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:48]: Same here

File: hello-world-office-addin/README.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+urlFragment: officedev-teamsfx-samples-hello-world-office-add-in
+description: A hello world project that contains Office add-in capability.
+extensions:
+  createdDate: "2024-12-06"
+---
+# Getting Started with Hello World Office Add-in Sample
+
+Office add-ins are integrations built by third parties into Office by using our web-based platform.
+
+Now you have the ability to create a single unit of distribution for all your Office applications by using the same manifest format and schema, based on the current JSON-formatted MetaOS manifest.
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:17]: With the unified app manifest, you can create a single unit of distribution for all your Microsoft 365 extensions by using the same manifest format and schema.

File: hello-world-office-addin/README.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+name: Hello World Office add-in
+urlFragment: officedev-teamsfx-samples-hello-world-office-add-in
+description: A hello world project that contains Office add-in capability.
+extensions:
+  createdDate: "2024-12-06"
+---
+# Getting Started with Hello World Office Add-in Sample
+
+Office add-ins are integrations built by third parties into Office by using our web-based platform.
+
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:16]:
```suggestion
```

File: hello-world-office-addin/README.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+Now you have the ability to create a single unit of distribution for all your Office applications by using the same manifest format and schema, based on the current JSON-formatted MetaOS manifest.
+
+Currently unified manifest of Office add-in is in public preview, so it does not support deploy and publish.
+
+## This sample illustrates
+
+- How to create an Office add-ins with a unified manifest that works on Word, Excel, Powerpoint, and Outlook.
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:23]:
```suggestion
- How to create an Office add-in with a unified manifest that works on Word, Excel, Powerpoint, and Outlook. 
```

File: hello-world-office-addin/README.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+   Once the Office app is open, you can select the ribbon command to show the add-in task pane. The snapshot shows Excel as an example, and the other Office applications share the same behavior. 
+   ![Excel add-in show taskpane](./assets/excel-ribbon.png)
+
+   The taskpane opens and you can check the sideloaded add-in.
+   ![Excel add-in task pane opened](./assets/thumbnail.png)
+
+
+### Edit the manifest
+
+You can find the app manifest in `./appPackage` folder. The folder contains one manifest file:
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:53]:
```suggestion
You can find the app manifest in `./appPackage` folder. The folder contains the manifest file `manifest.json` for Office add-in running locally.
```

File: hello-world-office-addin/README.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+
+- `manifest.json`: Manifest file for Office add-in running locally.
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:55]:
```suggestion

```

File: hello-world-office-addin/README.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+
+You may add any extra properties or permissions you require to this file. See the [schema reference](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema) for more information.
+
+
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:59]:
```suggestion
```

File: hello-world-office-addin/README.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+   ![Visual Studio Code debug configuration for Office](./assets/launch.png)
+
+   Once the Office app is open, you can select the ribbon command to show the add-in task pane. The snapshot shows Excel as an example, and the other Office applications share the same behavior. 
+   ![Excel add-in show taskpane](./assets/excel-ribbon.png)
+
+   The taskpane opens and you can check the sideloaded add-in.
+   ![Excel add-in task pane opened](./assets/thumbnail.png)
+
+
```
Comment (MingjiaLiu1995) [inline hello-world-office-addin/README.md:50]:
```suggestion
```

**Review by MingjiaLiu1995 — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**